From 52798a9d777321422353ca88b27249a6fb0473b3 Mon Sep 17 00:00:00 2001
From: SpiritCroc <dev@spiritcroc.de>
Date: Thu, 9 May 2024 09:21:21 +0200
Subject: Add fields for media caption support

---
 src/@types/media.ts | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/src/@types/media.ts b/src/@types/media.ts
index 1128b40d3..d85526642 100644
--- a/src/@types/media.ts
+++ b/src/@types/media.ts
@@ -189,6 +189,9 @@ export interface FileContent extends BaseContent {
      * One of: [m.file].
      */
     msgtype: MsgType.File;
+
+    format?: string;
+    formatted_body?: string;
 }
 
 /**
@@ -205,6 +208,9 @@ export interface ImageContent extends BaseContent {
      * One of: [m.image].
      */
     msgtype: MsgType.Image;
+
+    format?: string;
+    formatted_body?: string;
 }
 
 /**
@@ -221,6 +227,9 @@ export interface AudioContent extends BaseContent {
      * One of: [m.audio].
      */
     msgtype: MsgType.Audio;
+
+    format?: string;
+    formatted_body?: string;
 }
 
 /**
@@ -237,6 +246,9 @@ export interface VideoContent extends BaseContent {
      * One of: [m.video].
      */
     msgtype: MsgType.Video;
+
+    format?: string;
+    formatted_body?: string;
 }
 
 /**
-- 
2.46.2

