From c334d151d33ff1a761c9d8a76cae752953fdb5ad Mon Sep 17 00:00:00 2001
From: SpiritCroc <dev@spiritcroc.de>
Date: Sun, 27 Oct 2024 20:07:30 +0100
Subject: Schildify login footer

---
 src/components/views/auth/AuthFooter.tsx | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/components/views/auth/AuthFooter.tsx b/src/components/views/auth/AuthFooter.tsx
index 1942bf0431..1054d06f89 100644
--- a/src/components/views/auth/AuthFooter.tsx
+++ b/src/components/views/auth/AuthFooter.tsx
@@ -15,9 +15,13 @@ import { _t } from "../../../languageHandler";
 const AuthFooter = (): ReactElement => {
     const brandingConfig = SdkConfig.getObject("branding");
     const links = brandingConfig?.get("auth_footer_links") ?? [
+        /*
         { text: "Blog", url: "https://element.io/blog" },
         { text: "Mastodon", url: "https://mastodon.matrix.org/@Element" },
         { text: "GitHub", url: "https://github.com/element-hq/element-web" },
+        */
+        { text: "About", url: "https://schildi.chat" },
+        { text: "GitHub", url: "https://github.com/schildichat/schildichat-desktop" },
     ];
 
     const authFooterLinks: JSX.Element[] = [];
-- 
2.51.0

