From e92a08cfdf9d518c2f5a23b5cfdc67a0faa4dfdd Mon Sep 17 00:00:00 2001
From: SpiritCroc <dev@spiritcroc.de>
Date: Fri, 10 May 2024 18:58:32 +0200
Subject: Allow muted rooms to render as unread

---
 src/RoomNotifs.ts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/RoomNotifs.ts b/src/RoomNotifs.ts
index d5254d523d..9cb1e24d1d 100644
--- a/src/RoomNotifs.ts
+++ b/src/RoomNotifs.ts
@@ -255,7 +255,7 @@ export function determineUnreadState(
         return { symbol: "!", count: 1, level: NotificationLevel.Highlight, invited: false };
     }
 
-    if (getRoomNotifsState(room.client, room.roomId) === RoomNotifState.Mute) {
+    if (false && getRoomNotifsState(room.client, room.roomId) === RoomNotifState.Mute) {
         return { symbol: null, count: 0, level: NotificationLevel.None, invited: false };
     }
 
-- 
2.49.0

