From ee17d318c852cc119ff6357c4b7205a44fd5b1f8 Mon Sep 17 00:00:00 2001
From: SpiritCroc <dev@spiritcroc.de>
Date: Sun, 13 Oct 2024 11:23:54 +0200
Subject: Import SC compound theme overrides

---
 res/css/_common.pcss | 1 +
 1 file changed, 1 insertion(+)

diff --git a/res/css/_common.pcss b/res/css/_common.pcss
index caaa0df3b9..44c83ecd83 100644
--- a/res/css/_common.pcss
+++ b/res/css/_common.pcss
@@ -11,6 +11,7 @@ Please see LICENSE files in the repository root for full details.
 
 @import url("@vector-im/compound-design-tokens/assets/web/css/compound-design-tokens.css") layer(compound);
 @import url("@vector-im/compound-web/dist/style.css");
+@import "./sc-cpd-overrides.css";
 @import "./_font-sizes.pcss";
 @import "./_animations.pcss";
 @import "./_spacing.pcss";
-- 
2.50.1

