From dd6dd6d9e5b29fa4450c8be9527feec0ff0fe550 Mon Sep 17 00:00:00 2001
From: SpiritCroc <dev@spiritcroc.de>
Date: Sun, 13 Oct 2024 13:53:07 +0200
Subject: Always show timestamps by default

---
 src/settings/Settings.tsx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/settings/Settings.tsx b/src/settings/Settings.tsx
index 57fa5be09e..7de36fd0f7 100644
--- a/src/settings/Settings.tsx
+++ b/src/settings/Settings.tsx
@@ -821,7 +821,7 @@ export const SETTINGS: Settings = {
     "alwaysShowTimestamps": {
         supportedLevels: LEVELS_ACCOUNT_SETTINGS,
         displayName: _td("settings|always_show_message_timestamps"),
-        default: false,
+        default: true,
     },
     "userTimezone": {
         supportedLevels: LEVELS_DEVICE_ONLY_SETTINGS,
-- 
2.50.1

