From 66ac6916f3b57576711c0a3d71c2a7c133f4432b Mon Sep 17 00:00:00 2001
From: SpiritCroc <dev@spiritcroc.de>
Date: Thu, 14 Dec 2023 10:32:10 +0100
Subject: Don't preview reactions

---
 src/stores/room-list/MessagePreviewStore.ts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/stores/room-list/MessagePreviewStore.ts b/src/stores/room-list/MessagePreviewStore.ts
index e0e06ec980..7ee27f8402 100644
--- a/src/stores/room-list/MessagePreviewStore.ts
+++ b/src/stores/room-list/MessagePreviewStore.ts
@@ -57,10 +57,12 @@ const PREVIEWS: Record<
         isState: false,
         previewer: new StickerEventPreview(),
     },
+    /*
     "m.reaction": {
         isState: false,
         previewer: new ReactionEventPreview(),
     },
+    */
     [M_POLL_START.name]: {
         isState: false,
         previewer: new PollStartEventPreview(),
-- 
2.47.0

