From 2ffbb3cea351764ac33c347ba645262747bb962e Mon Sep 17 00:00:00 2001
From: SpiritCroc <dev@spiritcroc.de>
Date: Fri, 10 May 2024 18:58:32 +0200
Subject: Allow muted rooms to render as unread

---
 src/RoomNotifs.ts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/RoomNotifs.ts b/src/RoomNotifs.ts
index 1fb5e5ba4d..f9cb6ed026 100644
--- a/src/RoomNotifs.ts
+++ b/src/RoomNotifs.ts
@@ -263,7 +263,7 @@ export function determineUnreadState(
         return { symbol: "!", count: 1, level: NotificationLevel.Highlight };
     }
 
-    if (getRoomNotifsState(room.client, room.roomId) === RoomNotifState.Mute) {
+    if (false && getRoomNotifsState(room.client, room.roomId) === RoomNotifState.Mute) {
         return { symbol: null, count: 0, level: NotificationLevel.None };
     }
 
-- 
2.45.0

