From 19cacbc689d2cd27f400033c7c73dbe5fe9acf8d Mon Sep 17 00:00:00 2001
From: SpiritCroc <dev@spiritcroc.de>
Date: Tue, 9 Aug 2022 09:32:19 +0200
Subject: Exclude spaces from breadcrumbs

---
 src/stores/BreadcrumbsStore.ts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/stores/BreadcrumbsStore.ts b/src/stores/BreadcrumbsStore.ts
index 9859f24015..957b54286c 100644
--- a/src/stores/BreadcrumbsStore.ts
+++ b/src/stores/BreadcrumbsStore.ts
@@ -137,6 +137,7 @@ export class BreadcrumbsStore extends AsyncStoreWithClient<IState> {
     }
 
     private async appendRoom(room: Room): Promise<void> {
+        if (room.isSpaceRoom()) return;
         let updated = false;
         const rooms = (this.state.rooms || []).slice(); // cheap clone
         const msc3946ProcessDynamicPredecessor = SettingsStore.getValue("feature_dynamic_room_predecessors");
-- 
2.46.2

