From e87c6764f4abc734cdd05be97782e1b8e4e9df1c Mon Sep 17 00:00:00 2001
From: SpiritCroc <dev@spiritcroc.de>
Date: Sun, 13 Oct 2024 11:23:54 +0200
Subject: Import SC compound theme overrides

---
 .gitignore           | 2 ++
 res/css/_common.pcss | 1 +
 2 files changed, 3 insertions(+)

diff --git a/.gitignore b/.gitignore
index 3137cd555b..5345301b97 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,5 @@
+res/css/sc-*.css
+
 /.npmrc
 /*.log
 package-lock.json
diff --git a/res/css/_common.pcss b/res/css/_common.pcss
index 7861af6381..f7a660a339 100644
--- a/res/css/_common.pcss
+++ b/res/css/_common.pcss
@@ -11,6 +11,7 @@ Please see LICENSE files in the repository root for full details.
 
 @import url("@vector-im/compound-design-tokens/assets/web/css/compound-design-tokens.css") layer(compound);
 @import url("@vector-im/compound-web/dist/style.css");
+@import "./sc-cpd-overrides.css";
 @import "./_font-sizes.pcss";
 @import "./_animations.pcss";
 @import "./_spacing.pcss";
-- 
2.47.0

